integration-tests/mysql-client-tests/r/rmariadb-test.r

```diff
@@ -0,0 +1,65 @@
+library(RMariaDB)
+library(DBI)
+
+args = commandArgs(trailingOnly=TRUE)
+
+user = args[1]
+port = args[2]
+db = args[3]
+
+conn = dbConnect(RMariaDB::MariaDB(), host="127.0.0.1", port = port,
+                 username = user, dbname = db)
+
+# check standard queries
+queries = list("create table test (pk int, value int, primary key(pk))",
+               "describe test",
+               "insert into test (pk, value) values (0,0)",
+               "select * from test")
+
+responses = list(NULL,
+                 data.frame(Field = c("pk", "value"), Type = c("int", "int"), Null = c("NO", "YES"), Key = c("PRI", ""), Default = c("", ""), Extra = c("", "")),
+                 NULL,
+                 data.frame(pk = c(0), value = c(0)))
+
+for(i in 1:length(queries)) {
+    q = queries[[i]]
+    want = responses[[i]]
+    if (!is.null(want)) {
+        got <- dbGetQuery(conn, q)
+        if (!all(want == got)) {
+            print(q)
+            print(want)
+            print(got)
+            quit(1)
+        }
+    } else {
+        dbExecute(conn, q)
+    }
+}
+
+
+# check prepared statements
+statement <- enc2utf8(new("SQL", .Data = "INSERT INTO test (pk, value) values (?, ?)"))
```

Comment:
I think we should move to:

```r
stmt <- dbSendStatement(conn, 'INSERT INTO `test` VALUES (?, ?)')
rs <- dbBind(stmt, list(-16, '2020-09-10'))
rowsAff <- dbGetRowsAffected(rs)
dbClearResult(rs)
```

and do the same thing in the MySQL test as well.